src/uuid.cr

```diff
@@ -75,9 +75,9 @@ struct UUID
     new(uuid.bytes, variant, version)
   end
 
-  # Creates new UUID by decoding `value` string from hyphenated (ie. `ba714f86-cac6-42c7-8956-bcf5105e1b81`),
-  # hexstring (ie. `89370a4ab66440c8add39e06f2bb6af6`) or URN (ie. `urn:uuid:3f9eaf9e-cdb0-45cc-8ecb-0e5b2bfb0c20`)
-  # format.
+  # Creates new UUID by decoding `value` string from hyphenated (ie `ba714f86-cac6-42c7-8956-bcf5105e1b81`),
+  # hexstring (ie `89370a4ab66440c8add39e06f2bb6af6`) or URN (ie `urn:uuid:3f9eaf9e-cdb0-45cc-8ecb-0e5b2bfb0c20`)
+  # format, raising an `ArgumentError` if the string fors not match any of these formats.
```

Comment:
Added this distinction and removed the dots because they truncated the document string in the method index section in an unexpected spot:

<img width="570" alt="Truncated document string for UUID.new says: Creates new UUID by decoding value string from hyphenated (ie." src="https://user-images.githubusercontent.com/108205/163688135-e5beb18a-7327-42ae-8568-6e6bbbf58242.png">